static/styles/left-sidebar.scss

```diff
@@ -113,7 +113,7 @@ li.show-more-topics a {
     padding-top: 2px;
     padding-right: 0px;
     padding-bottom: 2px;
-    padding-left: $topic_indent;
+    padding-left: calc($topic_indent);
```

Comment:
Does it make sense to do the `calc` in the definition of `$topic_indent`?  Seems like potentially less of a land mine.

Reply:
I wasn’t sure if that would hypothetically prevent it from being used in a more complicated `calc` expression—but in fact `calc` expressions can be nested, so yeah, let’s do that.